its/ruling/src/test/resources/expected/python-S1192.json

```diff
@@ -1,43 +1,43 @@
 {
```

Comment:
Seeing the changes on the ruling, I'm not sure about whether this change is beneficial.

It seems that there are issues raised on strings that actually look like formatting patterns and I'm not sure we want that, despite the regex technically being fixed.
Wdyt?